src/main/java/app/linguistai/bmvp/service/AccountService.java

```diff
@@ -284,4 +284,18 @@ public boolean setPassword(String email, String password) throws Exception {
             throw new SomethingWentWrongException();
         }
     }
+
+    private void initiateUserSession(String email) throws Exception {
+        // Upon successful user entry, check whether to increase user streak or not
+        // ALSO IN MESSAGE SERVICE: userStreakService.updateUserStreak(dbUser.getEmail());
+        try {
+            userStreakService.updateUserStreak(email);
+        }
+        catch (Exception e1) {
+            // Intentionally not thrown to not cause login exception for users without UserStreak
+            System.out.println(ExceptionLogger.log(e1));
```

Comment:
I know this part came from the old call, however, can you add a simple `log` here instead of the `print`? Thanks!

Reply:
Yes changed it - though the streak service also calls log.error so it will be duplicate